playbooks/molecule/resources/xnat/inventory/group_vars/container_service.yml

```diff
@@ -6,3 +6,7 @@ docker_client_certificate_cache_directory:
 docker_server_hostname: "{{ hostvars['xnat_cserv']['hostname'] }}"
 docker_server_ip: "{{ hostvars['xnat_cserv']['ansible_ip'] }}"
 docker_server_port: 2376
+
+# docker<25 is required for XNAT
+# see: https://groups.google.com/g/xnat_discussion/c/yyPBkN4kayE/m/LUe5GQH5AAAJ
+docker_version: 24.0.9
```

Comment:
Nice, so the `docker` role uses `latest` but the `container_service` hosts are overridden to `24.0.9`?

Reply:
Should this be changed to https://github.com/UCL-MIRSG/ansible-collection-infra/blob/main/playbooks/molecule/resources/xnat/inventory/group_vars/container_service_host.yml? Currently, this would affect both `xnat_cserv` and `xnat_web`.

Reply:
And `xnat_web` doesn't use the `docker` role

Reply:
well spotted! Yep I'll move it to the container service hosts group